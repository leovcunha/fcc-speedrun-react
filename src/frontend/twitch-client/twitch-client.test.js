import React from "react";
import renderer from "react-test-renderer";
import TwitchClient from "../twitch-client";

describe("twitch client", () => {
    it("renders without crashing", () => {
        const component = renderer.create(<TwitchClient/>).toJSON();
        expect(component).toMatchSnapshot();
    });
});
