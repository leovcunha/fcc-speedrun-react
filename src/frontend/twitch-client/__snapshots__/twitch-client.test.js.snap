// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`twitch client renders without crashing 1`] = `
<div
  className="twitch-client jumbotron"
>
  <div
    className="row h1 justify-content-center"
  >
    Twitch Client
  </div>
  <hr
    className="p-5"
  />
</div>
`;
